JsonSchema/RequiredKeyword.cs

```diff
@@ -82,13 +82,7 @@ internal class RequiredKeywordJsonConverter : JsonConverter<RequiredKeyword>
 {
 	public override RequiredKeyword Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
 	{
-		using var document = JsonDocument.ParseValue(ref reader);
-
-		if (document.RootElement.ValueKind != JsonValueKind.Array)
-			throw new JsonException("Expected array");
-
-		return new RequiredKeyword(document.RootElement.EnumerateArray()
-			.Select(e => e.GetString()!));
+		return new RequiredKeyword(options.Read<IEnumerable<string>>(ref reader) ?? throw new JsonException("Expected array"));
```

Comment:
Is there a reason that you're using `IEnumerable` here instead of just an array?

Reply:
The `.Select` previously returned an `IEnumerable`; I matched previous types. However, the JsonSerializerOptions this way will determine the type; the default as determined by the .NET team to hopefully be the most performant option, which for now is probably still an array.

Reply:
> which for now is probably still an array.

... which is then put into a list in the constructor.

Reply:
I definitely didn't dig in that far! I'll switch that to an array so it uses the `params` constructor.